<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CASPER</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="geneEntryField">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_FindTargets">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Targets</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Damascus</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>CASPER</string>
    </property>
   </widget>
   <widget class="QLabel" name="casperDetail">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CRISPR Associated Software for Pathway Engineering and Research</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>841</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="versionNum">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>56</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>V 1.1</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_Gene">
       <property name="text">
        <string>Gene (Locus ID)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Position">
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Sequence">
       <property name="text">
        <string>Sequence</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Select Endonuclease:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="endoChoice"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_ViewTargets">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>View Targets</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>No Reference Genome</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>60</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Select Organism:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="orgChoice">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QRadioButton" name="Annotation_Kegg">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>17</height>
        </size>
       </property>
       <property name="text">
        <string>Kegg</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Annotation_Ownfile">
       <property name="text">
        <string>Choose Annotation fiile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="NCBI_Select">
       <property name="text">
        <string>NCBI</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Confused_Button">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>190</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>                       Annotations</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="Kegg_Search_Label">
       <property name="text">
        <string>Search KEGG Database for genes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="Kegg_Search_Imput"/>
       </item>
       <item>
        <widget class="QPushButton" name="Kegg_Search_Button">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="Own_File_Lable">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Select your own file</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="own_file_box">
       <item>
        <widget class="QLineEdit" name="Own_File_Imput">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Own_File_Button">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="NCBI_Label">
       <property name="text">
        <string>Use NCBI database</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLineEdit" name="NCBI_Imput">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="NCBI_Button">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="Annotations_Organism">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>320</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QCheckBox" name="Show_All_Results">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show all results </string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>20</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>420</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>180</y>
      <width>20</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="Add_Orgo_Combo">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_Orgo_Button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Organism</string>
    </property>
   </widget>
   <widget class="QLabel" name="Added_Org_Label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Added Organisms</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Added_Org_Combo">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Remove_Organism_Button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Org.</string>
    </property>
   </widget>
   <widget class="QLabel" name="Add_org_Label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>60</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Additonal Organisms</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Question_Button_add_org">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>60</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Recent"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionCut_Rev_Com"/>
    <addaction name="actionCopy_Rev_Com"/>
    <addaction name="actionPaste_Rev_Com"/>
   </widget>
   <widget class="QMenu" name="menuGenome">
    <property name="title">
     <string>Genome</string>
    </property>
    <addaction name="actionUpload_New_Genome"/>
    <addaction name="actionUpload_New_Endonuclease"/>
    <addaction name="actionChange_Directory"/>
    <addaction name="actionCo_Targeting"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuExport_to">
     <property name="title">
      <string>Export to...</string>
     </property>
     <addaction name="actionExcel_csv"/>
     <addaction name="actionIDT_Order"/>
    </widget>
    <addaction name="actionMultitargeting"/>
    <addaction name="actionPopulation_Analysis"/>
    <addaction name="actionComplete_Target_Analysis"/>
    <addaction name="separator"/>
    <addaction name="menuExport_to"/>
    <addaction name="actionNCBI_BLAST"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionNew_CASPER_Window"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuGenome"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpen_Recent">
   <property name="text">
    <string>Open Recent...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut_Rev_Com">
   <property name="text">
    <string>Cut Rev-Com</string>
   </property>
  </action>
  <action name="actionCopy_Rev_Com">
   <property name="text">
    <string>Copy Rev-Com</string>
   </property>
  </action>
  <action name="actionPaste_Rev_Com">
   <property name="text">
    <string>Paste Rev-Com</string>
   </property>
  </action>
  <action name="actionUpload_New_Genome">
   <property name="text">
    <string>Analyze New Genome</string>
   </property>
  </action>
  <action name="actionUpload_New_Endonuclease">
   <property name="text">
    <string>Upload New Endonuclease...</string>
   </property>
  </action>
  <action name="actionChange_Directory">
   <property name="text">
    <string>Change Directory</string>
   </property>
  </action>
  <action name="actionNCBI_BLAST">
   <property name="text">
    <string>NCBI BLAST</string>
   </property>
  </action>
  <action name="actionExcel_csv">
   <property name="text">
    <string>Excel (.csv)</string>
   </property>
  </action>
  <action name="actionIDT_Order">
   <property name="text">
    <string>IDT Order</string>
   </property>
  </action>
  <action name="actionMultitargeting">
   <property name="text">
    <string>Multitargeting</string>
   </property>
  </action>
  <action name="actionPopulation_Analysis">
   <property name="text">
    <string>Population Analysis</string>
   </property>
  </action>
  <action name="actionComplete_Target_Analysis">
   <property name="text">
    <string>Complete Target Analysis</string>
   </property>
  </action>
  <action name="actionNew_CASPER_Window">
   <property name="text">
    <string>New CASPER Window</string>
   </property>
  </action>
  <action name="actionCo_Targeting">
   <property name="text">
    <string>Co-Targeting</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
